src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -38,42 +20,46 @@ export const COLUMN_TITLES = [
   'user misery',
 ];
 
-export function generatePerformanceQuery(location: Location): Readonly<NewQuery> {
```

Comment:
This function was only used in tests, and I didn't think we needed it if we tested the resulting eventview which is more important in my opinion.